### ISSUE TITLE
Api fetch queue stops randomly

### ISSUE THREAD

**Issue by markusressel** [ekeih/InfiniteWisdom#21]
Sometimes the job that is sceduled to fetch new api images stops for no apparent reason. Altough this hasn't occurred for quite a while it should be investigated.

One thing that might improve the situation is to shorten the duration of this job as it probably never was meant to run such long running tasks.

**Comment by markusressel** [comment]
The suggestion in the second part of the description is done in #24 .
It also completely changes the way the crawling function is executed so this issue can probably closed when #24 is merged and the issue pops up again.

**Comment by markusressel** [comment]
Closed because of the reasons mentioned above.

**Comment by markusressel** [comment]
This issue recently popped up again.
The bot has been running for approx. XXX days XXX hours XXX minutes XXX seconds when image fetching suddenly stopped.

This is the last log message:
```
2019-08-01 04:44:37  DEBUG [infinitewisdom.util] Fetched image from: https://generated.inspirobot.me/a/Kj6ODq4ke0.jpg
```

No error was logged afterwards and the remaining functionality of the bot (like querying)  remained intact. The issue has been detected because after the bot was automatically restarted by kubernetes it suddenly started to download new images.

**Comment by markusressel** [comment]
Probably fixed with #52, closing.
